20_2 TreeList Single Row Selection: rewrite (#767)
DevExpress/devextreme-demos: JSDemos/Demos/TreeList/SingleRowSelection/description.md

> In this demo, the TreeList allows users to select only one row at a time. To enable this mode, set the **selection**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#mode) property to *"single"*. To select a row, just click it.

Reviewer: ```suggestion
In this demo, the TreeList allows you to select only one row at a time. To enable this mode, set the **selection**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#mode) property to *"single"*.
```
